stackslib/src/net/p2p.rs

```diff
@@ -105,7 +105,7 @@ struct NetworkHandleServer {
 
 impl NetworkHandle {
     pub fn new(chan_in: SyncSender<NetworkRequest>) -> NetworkHandle {
-        NetworkHandle { chan_in: chan_in }
+        NetworkHandle { chan_in }
```

Comment:
This is a pet peeve of mine -- can you limit your PR to only adding the new feature, instead of adding a bunch of extra line noise for needless code cleanup?  These changes do not improve code comprehensibility or fix bugs, but they do make the reviewer's job harder.

Reply:
I hope you don't mind that I just leave it for now. I know I also have a tendency for this as well and will refrain in the future.

Reply:
Yes, it's fine for now.